pkgs/flox-cli/default.nix

```diff
@@ -120,18 +117,16 @@
     # runtime dependencies of the dependent crates
     buildInputs =
       [
-        openssl.dev # octokit -> hyper -> ssl
-        zlib # git2
-        libssh2 # git2
-        libgit2 # git2
+        # reqwest -> hyper -> openssl-sys
+        openssl.dev
```

Comment:
> openssl.dev: hyper seems to build with rustls nowadays

We _could_ switch with:
```diff
diff --git a/cli/Cargo.toml b/cli/Cargo.toml
index d9694faa..e92c9b23 100644
--- a/cli/Cargo.toml
+++ b/cli/Cargo.toml
@@ -37,9 +37,9 @@ pollster = "0.3.0"
 progenitor-client = "0.6"
 proptest = "1.5.0"
 proptest-derive = "0.4.0"
-reqwest = { version = "0.11", features = ["json", "blocking"] }
+reqwest = { version = "0.11", default-features = false, features = ["json", "blocking", "rustls-tls"] }
 semver = "1.0.23"
-sentry = { version = "0.32.3", features = ["test"] }
+sentry = { version = "0.32.3", default-features = false, features = ["test", "rustls"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1"
 serde_with = "3.8.2"
```

Which removes the transient dep:
```console
% cargo tree --target all -i openssl
error: package ID specification `openssl` did not match any packages
```

But I have no idea what compatibility issues might arise on various platforms, so it doesn't seem worth doing without extra more planning/testing.

Reply:
yeah on macos it seems to either already use rustls or bind to the system openssl?
but then this was meant as a 1pt take out the trash not renovate the kitchen sink kind of effort.